### DISCUSSION TITLE
MathOpt CMake MathOpt tests

### DISCUSSION THREAD

**Discussion by sgatto** [google/or-tools#4391] [Build and configuration issues] [answered]
Hi,
At RTE we're just starting to work on an Xpress interface for MathOpt. We were wondering if it was possible to build and test using cmake. At the moment I'm struggling to even compile and link the unit tests `gscip_solver_test.cc`using CMake.

**Comment by Mizux** [comment, marked as answer]
on my way to do it, will be tracked in:
* #4402
